gui.py

```diff
@@ -284,34 +257,20 @@ def import_settings(self):
         """
         file_path = filedialog.askopenfilename(
                 parent=self.main_frame, title="Select Settings File",
-                filetypes=[("text files", "*.txt")])
+                filetypes=[("json files", "*.json")])

```

Comment:
```suggestion
            filetypes=[("JSON files (*.json)", "*.json"), ("All files", "*.*")]
```

Make file search options more consistent with Windows standards - having an "all files" option is always useful, just in case.